js/ad_manager_template.js

```diff
@@ -269,6 +269,20 @@ OO.Ads.manager(function(_, $) {
       // Stop any running ads
     };
 
+    /**
+     * Called by the Ad Manager Controller to determine if an ad video element must be created on player
+     * initialization. This is done so that the Video Controller can interface with the Ad's Video Plugin
+     * prior to an ad request. A typical use case would be to pass a user click to the ad plugin prior
+     * to ad playback so that the ad can start unmuted for browsers that require user interaction for
+     * unmuted playback.
+     * @method AdManager#createAdVideoElementOnPlayerInit
+     * @public
+     * @returns {string[]} An array of encoding types corresponding to the video elements that the Video Controller
+     *                     should create. Return an empty array, null, or undefined if this is not required.
+     */
+    this.createAdVideoElementOnPlayerInit = function() {
```

Comment:
Although it implicitly returns undefined, should we return an empty array?  Up to you.

Reply:
I've made a comment saying "Return an empty array, null, or undefined if this is not required." in the jsdoc. I think this is ok. What do you think?

Reply:
That's fine.

Reply:
Done